spec/policy_spec.lua

```diff
@@ -49,6 +49,56 @@ describe('policy', function()
     end)
   end)
 
+  describe('module equality', function()
+    it('equals when name and version are the same', function()
+      local p1 = policy.new('NAME', 'VERSION')
+      local p2 = policy.new('NAME', 'VERSION')
+
+      assert.are.equal(p1, p2)
+      assert.not_same(p1, p2)
+    end)
+
+    it('is not equal when names are different', function()
+      local p1 = policy.new('NAME', 'VERSION')
+      local p2 = policy.new('NAME2', 'VERSION')
+
+      assert.are.not_equal(p1, p2)
+    end)
+
+    it('is not equal when versions are different', function()
+      local p1 = policy.new('NAME', 'VERSION')
+      local p2 = policy.new('NAME', 'VERSION2')
+
+      assert.are.not_equal(p1, p2)
+    end)
+  end)
```

Comment:
Nitpick, just for completeness, maybe add a test comparing a policy against itself.

Reply:
Good one :)